### DISCUSSION TITLE
The domain analytics is dis-appear automatically.

### DISCUSSION THREAD

**Discussion by Booklifewealth** [vercel/community#5527] [Feature Question]
### Question

Hi, 

I have enabled an analytics for my domains hosting. They are disabled from 10 days back. 
please you check and help with this.

Thanks and Regards,
Booklifewealth

**Comment by Booklifewealth** [comment]
please can anyone from Vercel help me solve this issue?

**Comment by GavOutdoors** [comment]
I have the same issue.  Hardly any analytics for the last 6 days. 
I thought it might be when I moved over to the new Analytics tag. 

I have Cloudflare (which is reporting traffic). Cloudflare + Vercel Analytics worked previously, but maybe not with the new version of Vercel analytics. 
I added the cloudflare proxy bypass for the analytics url, but still very little on the Vercel dashboard.
